packages/shared/src/components/cards/CommentPopup.tsx

```diff
@@ -46,14 +42,8 @@ export default function CommentPopup({
     }
   };
 
-  let layoutModeClass: string;
-  if (listMode) {
-    layoutModeClass = 'h-full ml-14 justify-between';
-  } else if (compactCard) {
-    layoutModeClass = 'h-full';
-  } else {
-    layoutModeClass = 'h-3/4';
-  }
+  const layoutModeClass = 'h-3/4';
```

Comment:
This is now used only once in this file, I think we can now make this part of the inline classes.